doc/userguide/rule-management/rule-profiling.rst

```diff
@@ -0,0 +1,28 @@
+Rules Profiling
+===============
+
+If Suricata is built with the --enable-profiling-rules then the ruleset profiling
```

Comment:
(if there's a new version for this work)
nit: highlight the option more:
"with the `--enable-profiling-rules` option..."